### ISSUE TITLE
Using Tailwind with Direflow

### ISSUE THREAD

**Issue by AviusX** [Silind-Software/direflow#240]
So I have an existing React project that I now want to use with Direflow to convert it into a web component, but the project currently uses `craco` for the start, build and test scripts and uses TailwindCSS (setup as shown [here](https://tailwindcss.com/docs/guides/create-react-app)).
Now I can't use `craco` for the scripts since replacing `direflow-scripts` won't work. Using tailwind as a CDN disables some features I need. Is there a better way to do this with Direflow?

**Comment by LuisRodriguezLD** [comment]
what I personally do, is build the css using tailwind-cli before building my component.

example:
`NODE_ENV=production npx tailwindcss -i ./styles/globals.scss -o ./src/direflow-components/layout-component/style.css`

**Comment by hirenreshamwala** [comment]
You can use tailwind cli to compile the CSS and use output file in your app.js file

Add the following "script" in your package.json
```
"tw:watch": "npx tailwindcss -c ./tailwind.config.js -i ./src/direflow-components/<your-component-dir>/index.css -o ./src/direflow-components/<your-component-dir>/App.css --watch",
```

run command in another terminal
```
npm run tw:watch
```

import CSS file into your App.js
```
import styles from './App.css';
```

**Comment by AviusX** [comment]
I didn't reply earlier, but thank you both for responding. I've since moved on from the project where I was working on Direflow so I personally don't need to keep this issue open.

I do think that there should be a more hands off approach where we shouldn't manually have to run commands every time to make it work (maybe there is now, it's been a year since I opened this issue).

But in any case, this issue being open probably doesn't contribute to anything as of now, so I'll be closing it since this issue now at least has solutions available in the comments.
